pyomo/core/base/constraint.py

```diff
@@ -317,15 +337,22 @@ def expr(self):
 
     def get_value(self):
         """Get the expression on this constraint."""
-        return self._expr
+        return self.expr
 
     def set_value(self, expr):
         """Set the expression on this constraint."""
         # Clear any previously-cached normalized constraint
-        self._lower = self._upper = self._body = self._expr = None
-
+        self._expr = None
         if expr.__class__ in _known_relational_expressions:
+            if getattr(expr, 'strict', False) in _strict_relational_exprs:
+                raise ValueError(
+                    "Constraint '%s' encountered a strict "
+                    "inequality expression ('>' or '< '). All"
+                    " constraints must be formulated using "
```

Comment:
This doesn't matter and I won't bother you to change it but the inconsistent space at end / beginning of new lines irked me here.

Reply:
Fixed.